zcash_primitives/src/transaction/components/tze.rs

```diff
@@ -212,6 +212,11 @@ impl TzeOut {
     }
 }
 
+pub trait TzeInput {
```

Comment:
- Needs documentation (the trait and its methods).
- Document this addition in the changelog.